tests/unit/test_config.py

```diff
@@ -294,3 +294,33 @@ def test_configure_with_non_defined_handlers():
     }
     with pytest.raises(ValueError):
         dictConfig(config)
+
+def test_config_existing_disabled_logger_90195():
+    # See gh-90195
```

Comment:
Both the code and the test are basically straight from the CPython code. Is that okay license-wise? Should we bring in their copyright notices? (Current copyright only says MS)

Reply:
There is a copyright notice in the docs, the logging module also has a comment at the top about being copyright to the author, but considering the Python source code is distributed under a different license, I don't think it's relevant but I have made a reference to it to be sure.